### PR TITLE
Fixes SearchResults not displaying posts when searching

### PR DESCRIPTION
**Pull request by HyperTHD**
## Issue This PR Addresses
No issue was made for this PR

## Type of Change

- [x] **Bugfix**: Change which fixes an issue
- [ ] **New Feature**: Change which adds functionality
- [ ] **Documentation Update**: Change which improves documentation
- [ ] **UI**: Change which improves UI

## Description

This PR addresses the SearchResults page not properly displaying posts. This page originally used the telescopeURL, which was the original backend url used here. I switched the url to the new posts service url, which will correctly pull in the search microservice url. This bug was found when debugging #1910 and should hopefully help frontend developers working on the frontend search related components

## Checklist

- [x] **Quality**: This PR builds and passes our npm test and works locally
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not (if applicable)
- [ ] **Documentation**: This PR includes updated/added documentation to user exposed functionality or configuration variables are added/changed or an explanation of why it does not(if applicable)

### CODE REVIEW

File: src/web/src/components/SearchResults.tsx
Comment (humphd) [inline src/web/src/components/SearchResults.tsx:59]: A lot of our current config URLs deal with adding the end-point (i.e., `/query`), see https://github.com/Seneca-CDOT/telescope/blob/master/src/web/src/config.ts#L15-L18.  I wonder if we should do the same, so you'd do:

`${postsServiceUrl}?text=${encodeURIComponent(text)}&filter=${filter}&page=${index}`;

**Review by humphd — CHANGES_REQUESTED** [review]

File: src/web/src/components/SearchResults.tsx
Comment (humphd) [reply src/web/src/components/SearchResults.tsx:59]: Another question: why are we doing a search (/query) at the posts URL?

**Review by humphd — CHANGES_REQUESTED** [review]

**Comment by HyperTHD** [comment]
I done goofed up here. This PR should be switching to the ElasticSearch microservice url, not the posts microservice url. ELK is used here to index posts. This PR will have either be closed, or wait until #1802 is merged

**Comment by HyperTHD** [comment]
Filed #2014 to successfully link to this issue. This issue can also deal with moving the search service to production/staging since that'll also be needed for the frontend to work

**Comment by humphd** [comment]
You haven't dealt with my review comments, are you planning to fix?  If not, let's close this.

**Comment by humphd** [comment]
This is waiting on #1802.  We need the Search service in place before we can update this URL.

**Review by humphd — COMMENTED** [review]
Did your actual changes get lost here?

**Comment by HyperTHD** [comment]
> Did your actual changes get lost here?

They did, it's nothing minor since the change I made did not work, but I do need to fix the history here as well

**Comment by chrispinkney** [comment]
What's going on with this? Think we can get it for today's release?

**Pull request closed without merging**
